app/controllers/state_file/questions/verification_code_controller.rb

```diff
@@ -48,10 +48,19 @@ def update
       private
 
       def get_existing_intake(intake)
-        search = intake.class.where.not(id: intake.id, raw_direct_file_data: nil)
-        search = search.where(phone_number: intake.phone_number) if intake.phone_number.present?
-        search = search.where(email_address: intake.email_address) if intake.email_address.present?
-        search.first
+
+        state_intake_classes = StateFile::StateInformationService.state_intake_classes
```

Comment:
[dust] i'm guessing this is fine but do we care about NY? maybe we can exclude to save one lookup?

Reply:
I addressed this in commit `[704b9a0](https://github.com/codeforamerica/vita-min/pull/5463/commits/704b9a038d113e02c91fc7339f7663b12cf59773)`